### PR TITLE
Add Components

### PR DESCRIPTION
**Pull request by AkshithaReddy1899**
# This PR is created to add components

# Things to do

- Use React components.
- Use React props.
- Use React state.
- Use gitflow
- No linter errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import './styles/App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (oloomoses) [inline src/App.js]: - [ ] For this milestone it is required that you use class-based components as will have a chance to refactor them later. Kindly have a look at the screenshot below from the project requirements and make this a class-based component.
Screenshot from 2022-02-09 13-33-29

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import ButtonContainer from './ButtonContainer';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <Display />
+      <ButtonContainer />
+    </div>
+  );
+}
```
Comment (oloomoses) [inline src/components/Calculator.js]: - [ ] For this milestone it is required that you use class-based components as will have a chance to refactor them later. Kindly have a look at the screenshot below from the project requirements and make this a class-based component.
Screenshot from 2022-02-09 13-33-29

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @AkshithaReddy1899, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] Since you deleted the boilerplate file in the public directory, kindly delete their links in `index.html` file so as to avoid errors in the browser console.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Aksha, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AkshithaReddy1899** [comment]
Thank you

**Pull request merged**